url026791d51b_dhleong_SeriesTracker_v6_tgz-pJ8-stv6_SeriesTrackerJ8/src/stv6/handlers/ViewHandler.java

```diff
@@ -0,0 +1,106 @@
+package stv6.handlers;
+
+
+import stv6.Profile;
+import stv6.STServer;
+import stv6.episodes.SeriesEpisode;
+import stv6.http.request.Request;
+import stv6.http.request.RequestHandler;
+import stv6.series.TrackedSeries;
+import stv6.templating.Templator.Template;
+
+public class ViewHandler extends AbstractHandler implements RequestHandler {
+	
+	public static final String HANDLED_PAGE = "view";
+	public static final String TEMPLATE_NAME = "view.tpl";
+
+	@Override
+	public String getHandledPage() {
+		return HANDLED_PAGE;
+	}
+	
+	@Override
+	public String getTemplatePath() {
+		return TEMPLATE_NAME;
+	}
+
+	@Override
+	public boolean wrappedHandle(Request r, Template t) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 11. By comparison, 98% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test.

We recommend that you simplify this method or break it into multiple methods. For example, consider extracting the code block on lines 63-90 into a separate method.